ospd/ospd.py

```diff
@@ -200,7 +200,7 @@ class OSPDaemon(object):
       specific options eg. the w3af profile for w3af wrapper.
     """
 
-    def __init__(self, certfile, keyfile, cafile,
+    def __init__(self, certfile, keyfile, cafile, niceness,
```

Comment:
Personally I would make niceness optional
```suggestion
    def __init__(self, certfile, keyfile, cafile, niceness=None,
```